*** ./smali/android/view/LayoutInflater.smali	2015-10-30 17:01:46.243288787 +0100
--- /home/slimshadys/miui-v7/w7/temp/new_smali/framework.jar.out/./smali/android/view/LayoutInflater.smali	2015-10-30 16:57:16.983289551 +0100
***************
*** 1533,1538 ****
--- 1533,1546 ----
  
      move-result-object v1
  
+     invoke-static {v1, p1}, Landroid/view/LayoutInflaterMap;->getResourceId(Landroid/content/Context;I)I
+ 
+     move-result p1
+ 
+     invoke-virtual {p0}, Landroid/view/LayoutInflater;->getContext()Landroid/content/Context;
+ 
+     move-result-object v1
+ 
      invoke-virtual {v1}, Landroid/content/Context;->getResources()Landroid/content/res/Resources;
  
      move-result-object v1
