*** ./smali/android/net/wifi/SupplicantStateTracker.smali	2015-10-30 17:00:56.027288930 +0100
--- /home/slimshadys/miui-v7/w7/temp/new_smali/framework.jar.out/./smali/android/net/wifi/SupplicantStateTracker.smali	2015-10-30 16:56:19.971289712 +0100
***************
*** 24,30 ****
  
  .field private static final MAX_RETRIES_ON_ASSOCIATION_REJECT:I = 0x10
  
! .field private static final MAX_RETRIES_ON_AUTHENTICATION_FAILURE:I = 0x2
  
  .field private static final TAG:Ljava/lang/String; = "SupplicantStateTracker"
  
--- 24,30 ----
  
  .field private static final MAX_RETRIES_ON_ASSOCIATION_REJECT:I = 0x10
  
! .field private static final MAX_RETRIES_ON_AUTHENTICATION_FAILURE:I = 0x1
  
  .field private static final TAG:Ljava/lang/String; = "SupplicantStateTracker"
  
***************
*** 407,413 ****
  .end method
  
  .method private handleNetworkConnectionFailure(II)V
!     .locals 1
      .param p1, "netId"    # I
      .param p2, "disableReason"    # I
  
--- 407,413 ----
  .end method
  
  .method private handleNetworkConnectionFailure(II)V
!     .locals 2
      .param p1, "netId"    # I
      .param p2, "disableReason"    # I
  
***************
*** 429,434 ****
--- 429,444 ----
  
      invoke-virtual {v0, p1, p2}, Landroid/net/wifi/WifiConfigStore;->disableNetwork(II)Z
  
+     iget-object v0, p0, Landroid/net/wifi/SupplicantStateTracker;->mContext:Landroid/content/Context;
+ 
+     iget-object v1, p0, Landroid/net/wifi/SupplicantStateTracker;->mWifiConfigStore:Landroid/net/wifi/WifiConfigStore;
+ 
+     invoke-virtual {v1}, Landroid/net/wifi/WifiConfigStore;->getConfiguredNetworks()Ljava/util/List;
+ 
+     move-result-object v1
+ 
+     invoke-static {v0, v1, p1}, Landroid/net/wifi/SupplicantStateTrackerInjector;->handleNetworkConnectionFailure(Landroid/content/Context;Ljava/util/List;I)V
+ 
      return-void
  .end method
  
